contractcourt/htlc_incoming_contest_resolver.go

```diff
@@ -308,7 +308,8 @@ func (h *htlcIncomingContestResolver) Resolve(
 
 		resolution, err := h.Registry.NotifyExitHopHtlc(
 			h.htlc.RHash, h.htlc.Amt, h.htlcExpiry, currentHeight,
-			circuitKey, hodlQueue.ChanIn(), nil, payload,
+			circuitKey, hodlQueue.ChanIn(), h.htlc.CustomRecords,
```

Comment:
Hmm, I could swear I already had this diff in some version... But then didn't include it for some reason. Perhaps because it ended up working in a different way? Or because it ended up breaking something else? I guess we'll see about the latter in the itests.